src/components/SingleArticle.js

```diff
@@ -0,0 +1,70 @@
+import React, { useState, useEffect } from "react";
+import axios from "axios";
+import { View, Image, Text, StyleSheet } from "react-native";
+import { EBGaramond_400Regular } from "@expo-google-fonts/eb-garamond";
+
+const SingleArticle = ({ route }) => {
+  const { articleId } = route.params;
+  console.log(articleId);
```

Comment:
Is this intentional or leftover from debugging?

Reply:
leftover...